### ISSUE TITLE
Cannot find the setter for attribute 'app:items' with parameter type androidx.lifecycle.LiveData

### ISSUE THREAD

**Issue by maks** [evant/binding-collection-adapter#136]
## Issue
Following update from 3.0.0-beta1 to beta2 I now get a compile-time error:
```
****/ data binding error ****msg:Cannot find the setter for attribute 'app:items' with parameter type androidx.lifecycle.LiveData<java.util.List<my.data.class>> on androidx.recyclerview.widget.RecyclerView.
```
when trying to do binding for a recyclerview.

**Comment by evant** [comment]
What version of the Android Gradle plugin are you using?

**Comment by maks** [comment]
@evant I'm on Android Studio 3.2 rc2 that just came out so using the matching:
`classpath 'com.android.tools.build:gradle:3.2.0-rc02`

**Comment by evant** [comment]
Think I figured it out, made the the mistake of compiling it with the alpha gradle plugin `3.3.0-apha07`, seems that's not compatible with the `3.2.0-rc02`. Will publish an update.

**Comment by evant** [comment]
Should be fixed with `3.0.0-beta3`, let me know if you have any issues.
